introspection-engine/connectors/sql-introspection-connector/src/version_checker.rs

```diff
@@ -26,12 +27,14 @@ const SQLITE_TYPES: [(&'static str, &'static str); 5] = [
     ("TEXT", "TEXT"),
 ];
 
-const POSTGRES_TYPES: [(&'static str, &'static str); 5] = [
+const POSTGRES_TYPES: [(&'static str, &'static str); 7] = [
```

Comment:
nitpick: like @pimeys  mentioned, on slack, a `&[]` wouldn't need the explicit length in its type